src/services/api.ts

```diff
@@ -341,7 +363,12 @@ export const getInterestingVariants = (
   },
   signal?: AbortSignal
 ): Promise<InterestingVariantResult> => {
-  const endpoint = `/computed/find-interesting-variants?country=${country}`;
+  let endpoint = `/computed/find-interesting-variants?`;
+  if (isRegion(country)) {
+    endpoint += `country=Switzerland`;
```

Comment:
This should not be reachable, so this should throw. Defaulting to Switzerland could cause bugs in the future.